ios/RNCImageUtils.m

```diff
@@ -0,0 +1,20 @@
+//
+//  RNCImageUtils.m
+//  RNCImageEditor
+//
+//  Created by Dawid Urbaniak on 30/05/2019.
+//  Copyright © 2019 Facebook. All rights reserved.
+//
+
+#import "RNCImageUtils.h"
+
+@implementation RNCImageUtils
+
++ (id)writeImage:(id)image toPath:(id)path
+{
+    [image writeToFile:path atomically:YES];
```

Comment:
`writeToFile` returns YES if the operation succeeds, otherwise NO. Could you add check if creating finished with success?

Reply:
Done